frontend/test/playwright/visual-regression/components/global-audio-player.spec.ts

```diff
@@ -21,7 +22,10 @@ for (const dir of languageDirections) {
       )
       const audioRow = await audio.getNthAudioRow(page, 2)
       await audio.play(audioRow, dir)
-      await audio.pause(audioRow, dir)
+      await page
```

Comment:
Clicking the "pause" button on the search results sometimes doesn't wait for the audio to load, so the focus of the global audio track is not displayed correctly.
I changed it to click the "pause" button on the global audio player instead.